apps/mobile-mzima-client/src/app/map/map.page.html

```diff
@@ -2,7 +2,7 @@
   #layout
   [mode]="1"
   [offsetTop]="78"
-  [breakpoints]="[0, 300]"
+  [breakpoints]="[78, 300]"
```

Comment:
"Senior Intern Engineer" 🙌🏽 Nice figuring this part out @MMwandigha 

Although you scroll back down it bounces (goes hidden behind the menu) like once or twice before it now stays visible above the menu.

Maybe you want to make it more smooth, so that it just takes its place straight once scrolled down.

Reply:
😅Thanks  @Ifycode for checking out the PR.
For sure, let me have a look at it and see how to optimize it as stated

Reply:
@Ifycode Hmm I see. Do you suggest we remove the bouncing effect for both scenarios?

Reply:
Yes @MMwandigha